python/tests/test_store.py

```diff
@@ -16,21 +16,15 @@
 # under the License.
 
 import os
+
 import pytest
 
 from datafusion import SessionContext
-from datafusion.object_store import LocalFileSystem
-
-
-@pytest.fixture
-def local():
-    return LocalFileSystem()
```

Comment:
This isn't blocking for this PR but it appears we have very little test coverage for object store. It's a bit tricky, though, since I'm not sure where there are publicly available test resources that we could use.

Reply:
Indeed, we need more test coverage. I'll try to find some files to test it